locales/es.json

```diff
@@ -450,6 +452,8 @@
       "label": "Ajustes de cuenta"
     },
     "interface": {
+      "bottom_nav": "Navegación inferior",
+      "bottom_nav_instructions": "Elija sus botones de navegación favoritos, hasta cinco para la navegación inferior. Debe incluir el botón \"Otras opciones\".",
```

Comment:
We are translating in informal way according the other keys, so we can say here "Elige tus botones..." and "Debes incluir...". Wdyt?